apps/docs/content/docs/api-reference/openapi.yml

```
@@ -0,0 +1,314 @@
+            statusMessage: Internal Server Error
+            message: Unknown error
+            data:
+              url: https://github.com/techwithanirudh/coolify-tweaks/releases/download/latest/main.user.css
+              tag: latest
+
+tags:
+  - name: Releases
+    description: Endpoints for retrieving GitHub release assets with optional theme injection.
+  - name: Operations
+    description: Operational endpoints for monitoring and API information.
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)